src/basho_bench_stats_writer_csv.erl

```diff
@@ -129,24 +129,28 @@ report_global_stats({Op,_}, Stats, Errors, Units) ->
     %% insert correct units count
     JsonElements1 = lists:keyreplace(n, 1, JsonElements0, {'n', Units}),
 
+    %% insert correct ops count
+    JsonElements2 = lists:keyreplace(ops, 1, JsonElements1, {'ops', Operations}),
+
     %% insert error counts
-    JsonElements2 = [{basho_errors, Errors} | JsonElements1],
+    JsonElements3 = [{basho_errors, Errors} | JsonElements2]],
 
     JsonMetrics0 = erlang:get(run_metrics),
-    JsonMetrics = lists:keyreplace(Op, 1, JsonMetrics0, {Op, {JsonElements2}}),
+    JsonMetrics = lists:keyreplace(Op, 1, JsonMetrics0, {Op, {JsonElements3}}),
     %?DEBUG("Generated Json:\n~w",[JsonElements]),
     erlang:put(run_metrics, JsonMetrics).
 
 report_latency({_SummaryFile, _ErrorsFile},
                Elapsed, Window, Op,
-               Stats, Errors, Units) ->
+               Stats, Errors, Units, Operations) ->
     case proplists:get_value(n, Stats) > 0 of
         true ->
             P = proplists:get_value(percentile, Stats),
             Line = io_lib:format("~w, ~w, ~w, ~w, ~.1f, ~w, ~w, ~w, ~w, ~w, ~w\n",
                                  [Elapsed,
                                   Window,
                                   Units,
+                                  Operations,
```

Comment:
You're adding another parameter to the `io_lib:format` call above but not to the format string. You'll want to add another `~w` to the format string a few lines above.

Reply:
i missed it